openlibrary/plugins/openlibrary/status.py

```diff
@@ -25,8 +25,10 @@ def get_git_revision_short_hash():
             if status_info and isinstance(status_info, dict) 
             else None)
 
-def get_software_version():
-    return subprocess.Popen("git rev-parse --short HEAD --".split(), stdout = subprocess.PIPE, stderr = subprocess.STDOUT).stdout.read().strip()
+
+def get_software_version():  # -> str:
+    cmd = "git rev-parse --short HEAD --".split()
+    return str(Popen(cmd, stdout=PIPE, stderr=STDOUT).stdout.read().decode().strip())
```

Comment:
This is erroring on py3:

```
____ ERROR collecting openlibrary/plugins/importapi/tests/test_code_ils.py _____
openlibrary/plugins/importapi/tests/test_code_ils.py:2: in <module>
    from openlibrary.plugins.importapi import code
openlibrary/plugins/importapi/code.py:7: in <module>
    from openlibrary.plugins.openlibrary.code import can_write
openlibrary/plugins/openlibrary/code.py:936: in <module>
    setup()
openlibrary/plugins/openlibrary/code.py:923: in setup
    status.setup()
openlibrary/plugins/openlibrary/status.py:41: in setup
    version = version.decode("utf-8")
E   AttributeError: 'str' object has no attribute 'decode'
```

https://travis-ci.com/github/internetarchive/openlibrary/jobs/428085027#L610

Reply:
DONE